user/yaru-theme/template.py

```diff
@@ -0,0 +1,69 @@
+pkgname = "yaru-theme"
+pkgver = "24.10.4"
+pkgrel = 0
+build_style = "meson"
+hostmakedepends = [
+    "meson",
+    "pkgconf",
+    "bash",
+    "sassc",
+]
+makedepends = ["gtk+3-devel"]
+pkgdesc = "Default theme for ubuntu"
+maintainer = "metalparade <comer@live.cn>"
+license = "GPL-3.0-or-later OR CC-BY-SA-4.0"
+url = "https://github.com/ubuntu/yaru"
+source = f"{url}/archive/refs/tags/{pkgver}.tar.gz"
+sha256 = "80eed1a9832cf81976bf12b4cc85565b7d035176d1c3d73ccca7cacf1e4094a5"
+
+
+@subpackage("yaru-theme-gnome-shell")
+def _(self):
+    self.subdesc = "GNOME Shell"
+    return [
+        "usr/share/gnome-shell",
+        "usr/share/themes/*/index.theme",
+        "usr/share/themes/*/gnome-shell",
+    ]
+
+
+@subpackage("yaru-theme-gtk2")
+def _(self):
+    self.subdesc = "GTK2"
+    return ["usr/share/themes/*/gtk-2.0"]
```

Comment:
Chimera Linux doesn't have a GTK 2 package, so why even bother with this?

Reply:
emm you're right